src/code-templates/libraries/redis/redis-url-builder.ts

```diff
@@ -0,0 +1,10 @@
+import { RedisOptions } from './redis-client'
+
+export default (options: RedisOptions): string => {
```

Comment:
Maybe use named export? export default allows anyone to set any name, why not just use the function name?

maybe add a function name?

<img width="886" alt="image" src="https://user-images.githubusercontent.com/16746759/179224221-b55d43d6-5099-4a3f-a8a7-9cc66688f3c4.png">

[Source](https://github.com/goldbergyoni/nodebestpractices#-35-name-your-functions)